### PR TITLE
Moving program/levels/name into description

### PR DESCRIPTION
**Pull request by sharonwarner**
Connects #4024 

To test, do each of the following on UAT:
1) Load a worksheet with issues that are already saved in the database and confirm they're displayed correctly
2) Load a worksheet with issues that are not already saved in the database and confirm they're displayed correctly.

### CODE REVIEW

File: app/models/worksheet_issue.rb [hunk 1]
```diff
@@ -8,13 +8,31 @@ class WorksheetIssue < ActiveRecord::Base
...
         record.update(program: issue.program.try(:capitalize),
                       name: issue.type,
-                      levels: issue.description.join("; "),
+                      levels: issue.levels_with_codes.join("; "),
                       notes: issue.note,
+                      description: [
+                        [
+                          issue.program.try(:capitalize),
+                          issue.type
+                        ].compact.join(": ")
```
Comment (tejans24) [inline app/models/worksheet_issue.rb]: I think this and the logic in description can be turned into a common function

File: app/models/worksheet_issue.rb [hunk 2]
```diff
@@ -8,13 +8,31 @@ class WorksheetIssue < ActiveRecord::Base
...
+  def description
+    super || [[program, name].compact.join(": ")
+      .gsub(/Compensation/i, "Comp")
+      .gsub(/Service Connection/i, "SC")
+      .gsub(/Increased Rating/i, "IR"),
+              levels].compact.join("\n")
+  end
+
   class << self
     def create_from_issue(appeal, issue)
```
Comment (tejans24) [inline app/models/worksheet_issue.rb]: I think writing a unit test would be useful for this logic

Comment (sharonwarner) [reply app/models/worksheet_issue.rb]: Yeah, I'm definitely going to update tests... I'm just waiting on Travis.

File: app/models/worksheet_issue.rb (hunk 1 above)
Comment (sharonwarner) [reply app/models/worksheet_issue.rb]: Yeah, I could definitely break it out... but description is just a temporary method. As soon as the database migration goes in to update description, we'll delete it.

**Comment by sharonwarner** [comment]
Agreed about the generators... I'll do that in Part 7!

File: app/models/worksheet_issue.rb (hunk 2 above)
Comment (tejans24) [reply app/models/worksheet_issue.rb]: Travis has been terrible. Hopefully it starts working again normally soon.

**Pull request merged**